src/table.js

```diff
@@ -141,6 +141,11 @@ function isTypedArray(value) {
   );
 }
 
+export function isArqueroTable(value) {
+  // Arquero tables have a `toArrow` function
+  return typeof value.toArrow === "function";
```

Comment:
This will crash if value is null or undefined, rather than returning false as expected. We should add `value &&` here like we do for isArrowTable.

More importantly, though, our implementation doesn’t use _value_.toArrow—we use _value_.toArrowBuffer. And if we’re going to use _value_.toArrowBuffer, we should check that _value_.toArrowBuffer exists here rather than checking _value_.toArrow. Arguably, we could check both if we think there’s a risk of some other object implementing a _value_.toArrowBuffer method that does something different (_i.e._, that doesn’t return an Arrow IPC array buffer).

Reply:
Thank you, @mbostock! I should have caught all of those. Updated in https://github.com/observablehq/stdlib/pull/332/commits/705af99e5e8ab96c70589cb333ff352ab775e471 and https://github.com/observablehq/stdlib/pull/332/commits/7a84daf6ceab653cf5d126ef25b30d003d1a74a3.